cookbooks/bcpc/attributes/default.rb

```diff
@@ -106,9 +136,44 @@
     'http://s3.amazonaws.com/dev.hortonworks.com/HDP/ubuntu12/2.x/BUILDS/2.3.6.2-3'
 
   repos['hdp_utils'] =
-    'http://s3.amazonaws.com/dev.hortonworks.com/HDP-UTILS-1.1.0.20/repos/ubuntu12'
+    'http://public-repo-1.hortonworks.com/HDP-UTILS-1.1.0.20/repos/ubuntu12'
+
+  repos['cobbler26'] =
+    'http://download.opensuse.org/repositories/home:/libertas-ict:' \
+    '/cobbler26/xUbuntu_12.04/'
+
+  repos['zabbix'] = 'http://repo.zabbix.com/zabbix/2.4/ubuntu/'
+end
+
+default['bcpc']['repos_for']['trusty'].tap do |repos|
+  repos['rabbitmq'] = 'http://www.rabbitmq.com/debian'
+
+  repos['mysql'] = 'http://repo.percona.com/apt'
+
+  repos['hwraid'] = 'http://hwraid.le-vert.net/ubuntu'
+
+  repos['gridcentric'] =
+    'http://downloads.gridcentric.com/packages/%s/%s/ubuntu'
+
+  repos['ubuntu-tools'] =
+    'http://ppa.launchpad.net/canonical-support/support-tools/ubuntu'
+
+  repos['hortonworks'] =
+    'http://s3.amazonaws.com/dev.hortonworks.com/HDP/ubuntu14/2.x/BUILDS/2.3.6.2-3'
+
+  repos['hdp_utils'] =
+    'http://public-repo-1.hortonworks.com/HDP-UTILS-1.1.0.20/repos/ubuntu14'
+
+  repos['cobbler26'] =
+    'http://download.opensuse.org/repositories/home:/libertas-ict:' \
+    '/cobbler26/xUbuntu_14.04'
+
+  repos['zabbix'] = 'http://repo.zabbix.com/zabbix/2.4/ubuntu/'
```

Comment:
I experimented with ways to avoid duplicates, but I concluded it would be too annoying to try to override in an environment file if I made the scheme more complex.